pyproject.toml

```diff
@@ -30,6 +30,7 @@ exclude = [
 ]
 line-length = 165
 fix = true
+target-version = "py39"
```

Comment:
This is not needed, as Ruff is smart enough to use `project.requires-python`, see https://docs.astral.sh/ruff/settings/#target-version